src/Ionide.ProjInfo/Library.fs

```diff
@@ -459,6 +536,9 @@ module ProjectLoader =
             "DotnetProjInfo", "true"
             yield! globalProperties
         ]
+        |> List.filter (fun (ourProp, _) -> not (propsSetFromParentCollection.Contains ourProp))
```

Comment:
the only reason that this was set is because `Configuration` is set by default here but some of our tests set `Release` and expect that to work. 

If we removed this default it would be cleaner IMO, plus the SDK already defaults to Debug. Happy to discuss for a different PR.